docs/conf.py

```diff
@@ -17,15 +17,9 @@
 # documentation root, use os.path.abspath to make it absolute, like shown here.
 #
 
-import os
-import sys
 from datetime import datetime
 
 
-# append the ddtrace path to syspath
-sys.path.insert(0, os.path.abspath(".."))
```

Comment:
why no longer needed?

Reply:
See the end of https://stackoverflow.com/a/31882049 - it breaks the building of docs :confused:

Reply:
fun